### PR TITLE
Feature/docker compose

### PR DESCRIPTION
**Pull request by wshino**
#245 

- [x] connect deployer -> ganache-cli
- [x] set deployer tasks to Dockerfile without to connect ganache-cli
- [x] run operator with dockerhub image
- [x] set deployer build to multistage build
- [x] install curl into operator
- [x] connect wallet -> operator
- [x] set target constant to variable in container
- [x] remove unused lines

### CODE REVIEW

**Comment by syuhei176** [comment]
Thank you for your great work! I can connect to root chain with cli.

Some modification to be needed.

- better to use .plasmadb instead of plasmadb
- use latest child chain(will make another issue)

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  operator:
+    build:
+      context: ./
+      dockerfile: ./Dockerfile
+    image: "cryptoeconomicslab/plasma-chamber:development"
+    ports:
+      - "3000:3000"
+    entrypoint: sh ./wait.sh /home/node/.npm-global/bin/layer2-operator
+    volumes:
+      - ./plsmadb:/var/plasmadb
```
Comment (syuhei176) [inline docker-compose.yaml]: better to specify `.plasmadb`.

```
      - ./.plasmadb:/var/plasmadb
```

**Comment by yuriko627** [comment]
Thanks so much Shinoharasan! 💛

**Comment by wshino** [comment]
This compose yaml file is for operation check. Next PR, I will add container settings for develop. But, npm build in Docker is not suitable for macOS, because hyperkit daemon reaches the limit of its cpu that is related to disk IO between host and container.

**Pull request merged**